### ISSUE TITLE
Add limiting feature

### ISSUE THREAD

**Issue by jabbink** [jabbink/PokemonGoBot#1124]
**Description:**
Automatically disable certain features of the bot when the ban is near.
When there are no more eggs (so walking makes no sense at all), disable the bot completely and restart it after some time.

Must save last location #144 

Leave this sentence in your issue as proof that you have read and used this template.

**Comment by MacDogg** [comment]
But you get an egg from hitting a Pokestop so it kinda does make sense to walk.  I have never ran out of eggs on any account and they all have 9 right now.  As soon as one hatches and I drop in another I get a new egg very quickly.

**Comment by yashendra2797** [comment]
I'm stuck at 9 eggs. They haven't hatched since the last 2 releases or so. The breakable incubators are used, but the infinite one isn't.

**Comment by muschter** [comment]
@MacDogg Yes it does, but if you hit the Pokestop threshold that I submitted a PR for, you won't be getting any more eggs.

**Comment by chongivan** [comment]
@jabbink, how do you define when "the ban is near"?
